app/src/androidTest/java/org/mozilla/focus/activity/HomeTest.java

```diff
@@ -124,4 +128,41 @@ public void checkBasicHomeComponents_allAreVisible() {
 
     }
 
+    /**
+     * Test case no: TC0039
+     * Test case name: Dismiss menu
+     * Steps:
+     * 1. Launch app
+     * 2. Tap menu
+     * 3. Press back key
+     * 4. Check menu panel not exist
+     * 5. Tap menu
+     * 6. Tap on blank space above menu or
+     * 7. Check menu panel not exist
+     */
+    @Test
+    public void dismissMenu() {
+        activityRule.launchActivity(new Intent());
+
+        // Tap menu
+        AndroidTestUtils.tapHomeMenuButton();
+
+        // Press back
+        Espresso.pressBack();
+
+        // Check menu panel not exist
+        onView(withId(R.id.menu)).check(doesNotExist());
+
+        // Tap menu
+        AndroidTestUtils.tapHomeMenuButton();
+
+        // Tap on blank space above menu
+        final DisplayMetrics displayMetrics = activityRule.getActivity().getResources().getDisplayMetrics();
+        final int displayWidth = displayMetrics.widthPixels;
+        final int displayHeight = displayMetrics.heightPixels;
+        onView(withId(R.id.menu)).perform(AndroidTestUtils.clickXY(displayWidth / 2, -displayHeight / 2, Tap.SINGLE));
```

Comment:
is there a better way to check this?
I imagine this might not always be the case.

Reply:
@cnevinc  will help on finding suitable element

Reply:
Maybe we can click on its parent and dismiss that dialogue fragment. Don't have time to look at it now.